README.md

```diff
@@ -16,14 +16,14 @@ Dredd uses the [Gavel][] for judging if a particular API response is valid
 or if it isn't. If you are curious about how decisions are made, please refer
 to Gavel's [behavior specification][].
 
-Dredd supports writing [hooks](http://dredd.readthedocs.org/en/latest/hooks-ruby/hooks/) glue code for testing statefull servers, workflows and loading fixtures in:
+Dredd supports writing [hooks](http://dredd.readthedocs.org/en/latest/hooks/) glue code for testing statefull servers, workflows and loading fixtures in:
 
 - [Ruby](http://dredd.readthedocs.org/en/latest/hooks-ruby/)
-- [Python](http://dredd.readthedocs.org/en/latest/hooks-ruby/hooks-python/)
-- [Node.js](http://dredd.readthedocs.org/en/latest/hooks-ruby/hooks-node/)
+- [Python](http://dredd.readthedocs.org/en/latest/hooks-python/)
+- [Node.js](http://dredd.readthedocs.org/en/latest/hooks-node/)
```

Comment:
http://dredd.readthedocs.org/en/latest/hooks-node/ is a dead link.  It should be URL: http://dredd.readthedocs.org/en/latest/hooks-nodejs/ 

Also, hooks.md references hooks-node and should be updated to hooks-nodejs